test/e2eTests/tests/registerPatient_test.js

```diff
@@ -22,28 +39,250 @@ Scenario('Validate tab sequence', (I, RegistrationDashboardPage) => {
   I.say(`${LOG_TAG} Click on the New Patient button`);
   registrationPage.clickNewPatienButton();
```

Comment:
This function should return the registerPatientPage. See example [here](https://github.com/esaude/esaude-emr-poc/blob/master/test/e2eTests/pages/loginPage.js#L38)

Reply:
It navigates to that page. So, yes.